frontend/src/components/Navbar/index.tsx

```diff
@@ -8,15 +8,44 @@ export default function Navbar() {
   const theme = useTheme();
   const {isAuth, user} = useAuthStore();
   return (
-    <Box component="nav" className={styles.header}>
-      <Typography variant="h1" color={theme.palette.primary.main}>
+    <Box
+      component="nav"
+      style={{
+        display: 'flex',
+        justifyContent: 'space-between',
+      }}
+    >
+      <Typography
+        variant="h1"
+        color={theme.palette.primary.main}
+        style={{fontSize: 60}}
+      >
         Busify
       </Typography>
-      <Typography>A centrailed bus ticket booking system</Typography>
+      <Typography style={{marginTop: '1.5rem', fontSize: 20}}>
+        A centrailed bus ticket booking system
+      </Typography>
       {isAuth ? (
-        <Typography>Hi! {user?.name}</Typography>
+        <div>
+          <Typography
+            style={{fontSize: 40, color: 'rgba(214, 202, 202, 0.788)'}}
+          >
+            Hi! {user?.name}
+          </Typography>
+          <img
+            src=""
+            alt="User Image"
+            style={{width: 30, height: 30, borderRadius: 15}}
+          ></img>
+        </div>
       ) : (
-        <Button variant="contained" color="primary" href={getGoogleOAuthURL()}>
+        <Button
+          variant="contained"
+          color="primary"
+          href={getGoogleOAuthURL()}
+          style={{height: '60px', marginTop: '0.4rem'}}
+        >
+          <img src="google_icon.png" style={{height: '40px'}} />
```

Comment:
Don't add it here, pass it as a prop without any styling with the above button as `startIcon`.